tensorflow_datasets/core/download/download_manager.py

```diff
@@ -519,9 +519,10 @@ def download_kaggle_data(self, competition_name):
     with self._downloader.tqdm():
       kaggle_downloader = self._downloader.kaggle_downloader(competition_name)
       urls = kaggle_downloader.competition_urls
-      files = kaggle_downloader.competition_files
-      return _map_promise(self._download,
-                          dict((f, u) for (f, u) in zip(files, urls)))
+      # files = kaggle_downloader.competition_files
```

Comment:
Please cleanup the code

Reply:
Yeah working on it right now. I just wanted to make sure all the issues were completely fixed before removing comments.